packages/@lwc/engine-server/src/__tests__/fixtures/attribute-aria/index.js

```diff
@@ -1,2 +1,3 @@
 export const tagName = 'x-parent';
 export { default } from 'x/parent';
+export * from 'x/parent';
```

Comment:
In the Rollup.js world, `export *` does not include `default`. This is actually not spec compliant, but is a strangeness that is called out in the docs explicitly.

However, since these fixtures are being re-used to test `@lwc/ssr-compiler`, and because these new compiled artifacts now include named exports (e.g. `generateMarkup`), I've modified the test fixtures to work for both `@lwc/engine-server` and `@lwc/ssr-compiler`.